go_to_office/lib/screens/admin/offices_list.dart

```diff
@@ -11,18 +13,28 @@ final List<Office> officesList = [
 ];
 
 class OfficesListPage extends StatefulWidget {
+  OfficesListPage({Key key, this.title, this.repository, this.adminName}) : super(key: key);
+
+  final String title;
+  final Repository repository;
+  final String adminName;
+
   @override
-  State<StatefulWidget> createState() => _OfficesListPageState();
+  State<StatefulWidget> createState() => _OfficesListPageState(repository, adminName);
 }
 
 class _OfficesListPageState extends State<OfficesListPage> {
+  _OfficesListPageState(this.repository, this.adminName);
+
+  final Repository repository;
+  final String adminName;
+
   @override
   Widget build(BuildContext context) {
     // Object officateAssreddress = {};
     return Scaffold(
-        appBar: AppBar(
-          title: Text(Strings.offices),
-        ),
+        appBar: AppBar(title: Text(widget.title)),
+        drawer: NavDrawer(userType: Strings.loginAdmin, title: adminName, repository: repository),
```

Comment:
Do we need to pass to the NavDrawer the userType and title?
Can't the NavDrawer know using the repository that the user which is logged in is an admin user and what is his name?

Reply:
It is temporary.
Because we still don't have the repo fully implemented (getting the current user type etc... ).

Reply:
Are you sure we don't have the ability to get from the repository the current user?